feat(Apple): reportAccessibilityIdentifier option (#10923)
getsentry/sentry-docs: docs/platforms/apple/common/enriching-events/viewhierarchy/index.mdx

<PlatformContent includePath="enriching-events/attach-viewhierarchy" />

> ## Privacy within View Hierarchy Attachments
> 
> Sentry does not collect any user information with the view hierarchy, such as UILabel and UITextView text. The view hierarchy attachment only contains the view hierarchy structure and properties.
> 
> Some developers use the view's accessibilityIdentifier property with personal information. If you are using this property, make sure to disable accessibilityIdentifier reporting like this:

Reviewer: ```suggestion
## Privacy In View Hierarchy Attachments

Sentry doesn't collect any user information (such as UILabel and UITextView text) with the view hierarchy attachment, which only contains structure and properties.

If you're using the view's `accessibilityIdentifier` property with personal information, make sure to disable `accessibilityIdentifier` reporting like in the example below:
```
